assets/collapsible-content.css

```diff
@@ -10,8 +10,9 @@
   }
 }
 
-.collapsible-content__media {
-  background-color: transparent;
+.collapsible-content {
+  position: relative;
+  z-index: 0;
```

Comment:
Should this be using `isolate` cc @sofiamatulis ?

Reply:
It could be I figured i'd use what is currently being used on the theme at this moment.

Reply:
We can re-use the class `.isolate` that I have added on my PR https://github.com/Shopify/dawn/pull/1209